lookout/style/format/tests/test_quality_report.py

```diff
@@ -84,73 +84,70 @@ def test_eval_empty_input(self):
         input_pattern = os.path.join(self.jquery_dir, "**", "*")
         with Capturing() as output:
             print_reports(input_pattern=input_pattern, bblfsh=self.bblfsh,
                           language=self.language, model_path=self.model_path,
                           config={"uast_break_check": False})
-        self.assertEqual([
-            "# Quality report for javascript / https://github.com/jquery/jquery refs/heads/master"
-            " c2026b117d1ca5b2e42a52c7e2a8ae8988cf0d4b",
-            "",
-            "### Classification report"],
-            output[:3])
-        qcount = output.count(
-            "# Quality report for javascript / https://github.com/jquery/jquery refs/heads/master"
-            " c2026b117d1ca5b2e42a52c7e2a8ae8988cf0d4b")
-        self.assertEqual(qcount, 14)
+        self.assertIn(q_report_header, output[0])
         self.assertIn("### Summary", output)
-        self.assertIn("# Model report for https://github.com/jquery/jquery refs/heads/master"
-                      " c2026b117d1ca5b2e42a52c7e2a8ae8988cf0d4b",
-                      output)
+        self.assertIn("### Classification report", output)
         self.assertGreater(len(output), 100)
-        self.assertIn("javascript", _get_json_data("\n".join(output)))
+        output = "\n".join(output)
+        self.assertIn("javascript", _get_json_data(output))
+        self.assertIn("# Model report", output)
+        qcount = output.count(q_report_header)
+        self.assertEqual(qcount, 14)
 
     def test_eval_aggregate(self):
         """Test on normal input, quality reports are aggregated."""
+        q_report_header = "# Quality report for javascript"
         input_pattern = os.path.join(self.jquery_dir, "**", "*")
         with Capturing() as output:
             print_reports(input_pattern=input_pattern, bblfsh=self.bblfsh,
                           language=self.language, model_path=self.model_path,
                           config={"uast_break_check": False, "aggregate": True})
-        qcount = output.count(
-            "# Quality report for javascript / https://github.com/jquery/jquery refs/heads/master"
-            " c2026b117d1ca5b2e42a52c7e2a8ae8988cf0d4b")
-        self.assertEqual(qcount, 1)
         output = "\n".join(output)
-        output = output[:output.find("# Model report for https://github.com/jquery/jquery")]
+        qcount = output.count(q_report_header)
+        self.assertEqual(qcount, 1)
+        output = output[:output.find("# Model report for")]
         metrics = _get_metrics(output)
-        expected_metrics = Metrics(precision=0.9829633453794527, ppcr=0.6570556309362280,
-                                   recall=0.9829633453794527, full_recall=0.6458616010854816,
-                                   f1=0.9829633453794527, full_f1=0.7795291709314227,
-                                   support=1937, full_support=2948)
+        expected_metrics = (9.756371814092953e-01, 9.756371814092953e-01,
```

Comment:
Can you please remove the scientific notation here - it is very hard to comprehend